docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
+NVIDIA GPU. This is achieved by configuring the underlying MicroK8s, that the
```

Comment:
```suggestion
NVIDIA GPU. You can do so by configuring the underlying MicroK8s, on which DSS rely on for running the containerised workloads.
```

Reply:
Ah, just saw the second part of the suggestion

pushed a commit 5a8fba4

(the suggestions are now outdated and I can't apply some of them...)